test/parallel/test-process-kill-pid.js

```diff
@@ -68,6 +68,16 @@ const unknownSignal = common.expectsError({
 assert.throws(function() { process.kill(1, 'test'); },
               unknownSignal);
 
+// Test that kill throws an error for invalid signal numbers
+const invalidSignalNumber = common.expectsError({
+  code: 'EINVAL',
+  type: Error,
+  message: 'kill EINVAL'
+});
+
+assert.throws(function() { process.kill(1, 987); },
+              invalidSignalNumber);
```

Comment:
this can be simplified to just

```js
common.expectsError(
  () => process.kill(1, 987),
  {
    code: 'EINVAL',
    type: Error,
    message: 'kill EINVAL'
  }
);
```

Reply:
done